lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -47,7 +51,8 @@ def start
 
   def client
     @_es ||= begin
-      adapter_conf = lambda {|f| f.adapter :excon }
+      excon_options = { client_key: @client_key, client_cert: @client_cert, client_key_pass: @client_key_pass }
+      adapter_conf = lambda {|f| f.adapter :excon, excon_options }
```

Comment:
@ewolinetz It's kind of normal for people to use keys without passwords in non-interactive apps sadly so using Faraday should cover most use cases IMO. But there will be others who do need a key password so...